pkg/scte35/pids.go

```diff
@@ -0,0 +1,49 @@
+package scte35
+
+//Pids holds collections of pids by type for scte35.Stream.
+type Pids struct {
```

Comment:
We prefer acronyms and initialism be all caps or all lower case, whether used as a var or part of a function name (so `PIDs` is preferred to `Pids`)